src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -49,14 +49,15 @@ export const Reaction = styled.div<ReactionProps>`
 
 // --- Reaction Container
 export const Container = styled.div`
-  // @debt convert this to scss then use our z-index layer helper here
-  z-index: 11;
```

Comment:
I believe all of the other s-indexes should be removed from here as well

Reply:
You are right.

Removed in https://github.com/sparkletown/sparkle/pull/1324/commits/6d727cdede703fa438334cfb513f48665d1dbe71